apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,93 @@
+html {
+    box-sizing: border-box;
+  }
+  *, *:before, *:after {
+    box-sizing: inherit;
+  }
+  
+  body {
+    margin: 0;
+    padding: 0;
+  }
+
+body {
+    color: #111;
+    font-family: Helvetica, Arial, sans-serif;
+    font-size: 16px;
+    line-height: 147%;
+}
+
+img { 
+    display: block;
+    max-width: 100%; 
+    margin: 0 auto;
+}
+
+hr {
```

Comment:
`hr` make you a horizontal line. So you shouldn't use `border` property here. 
You can use either `hr` without border or a `div` with border property.